Content.Server/Cloning/CloningSystem.cs

```diff
@@ -113,6 +93,49 @@ public bool TryCloning(EntityUid original, MapCoordinates? coords, ProtoId<Cloni
         return true;
     }
 
+    /// <summary>
+    /// Clone components based on a cloneSetting list
+    /// </summary>
+    /// <param name="original">The orignal Entity to Clone components from</param>
+    /// <param name="clone">the target Entity to clone Components too</param>
+    /// <param name="settings">the Clone settings list of components to clone</param>
+    public void CloneComponents(EntityUid original, EntityUid clone, CloningSettingsPrototype settings)
+    {
+        var componentsToCopy = settings.Components;
+        var componentsToEvent = settings.EventComponents;
+        // don't make status effects permanent
+        if (TryComp<StatusEffectsComponent>(original, out var statusComp))
+            componentsToCopy.ExceptWith(statusComp.ActiveEffects.Values.Select(s => s.RelevantComponent).Where(s => s != null)!);
```

Comment:
```suggestion
        if (TryComp<StatusEffectsComponent>(original, out var statusComp))
        {
            var statusComps = statusComp.ActiveEffects.Values.Select(s => s.RelevantComponent).Where(s => s != null);
            componentsToCopy.ExceptWith(statusComps!);
            componentsToEvent.ExceptWith(statusComps!);
        }
```
best do this for both lists